lldb/include/lldb/API/SBDebugger.h

```diff
@@ -321,8 +321,15 @@ class LLDB_API SBDebugger {
 
   void SetLoggingCallback(lldb::LogOutputCallback log_callback, void *baton);
 
-  void SetDestroyCallback(lldb::SBDebuggerDestroyCallback destroy_callback,
-                          void *baton);
+  lldb::SBDebuggerDestroyCallbackToken
+  AddDestroyCallback(lldb::SBDebuggerDestroyCallback destroy_callback,
+                     void *baton);
+
+  lldb::SBDebuggerDestroyCallbackToken
+  SetDestroyCallback(lldb::SBDebuggerDestroyCallback destroy_callback,
```

Comment:
This is also deprecated.  I'd add the same comment here, and use the LLDB_DEPRECATED_IN macro to mark it formally.

Reply:
Formally, returning the token from SetDestroyCallback is a reasonable addition, but it changes the signature of an extant SB API, which we don't allow - we are strict about maintaining backwards compatibility of the SB API's.  Since we're deprecating this API anyway, I don't think we need to make Set play well with the Add/Remove model.  Just leave it returning void.

Reply:
Yeah. Thanks for catching this. Reverted the API changes (both in SBDebugger and in Debugger).